components/pages/Vote.tsx

```diff
@@ -1,11 +1,37 @@
-import { Banner, HowItWorks, Layout, Votes } from "components";
+import { Banner, HowItWorks, Layout, LoadingSpinner, PageInnerWrapper, PageOuterWrapper, Votes } from "components";
+import { useVotesContext } from "hooks";
+import styled from "styled-components";
 
 export function Vote() {
+  const { getUserIndependentIsLoading } = useVotesContext();
   return (
     <Layout>
-      <Banner />
+      <Banner>
```

Comment:
Moved the markup for the layout here from the `Votes` component so that this pattern matches the one in `WalletSettings`